image/db/rhel/konflux.Dockerfile

```
@@ -50,3 +44,22 @@ EXPOSE 5432
 CMD ["postgres", "-c", "config_file=/etc/postgresql.conf"]
 
 USER 70:70
+
+FROM scanner-db-common as scanner-db-slim
+
+LABEL \
+    com.redhat.component="rhacs-scanner-db-slim-container" \
+    io.k8s.display-name="scanner-db-slim" \
+    name="rhacs-scanner-db-slim-rhel8"
+
+ENV ROX_SLIM_MODE="true"
+
+FROM scanner-db-common as scanner-db
+
+LABEL \
+    com.redhat.component="rhacs-scanner-db-container" \
+    io.k8s.display-name="scanner-db" \
+    name="rhacs-scanner-db-rhel8"
+
+COPY blob-pg-definitions.sql.gz \
+     /docker-entrypoint-initdb.d/definitions.sql.gz
```

Well, you can keep `root` ownership by doing this:

```suggestion
USER 0:0
COPY blob-pg-definitions.sql.gz \
     /docker-entrypoint-initdb.d/definitions.sql.gz
USER 70:70
```

---

Is it necessary though or was root ownership just an artifact of the midstream Dockerfile?